yarn-project/simulator/src/avm/avm_simulator.test.ts

```diff
@@ -59,6 +59,24 @@ describe('AVM simulator', () => {
       expect(results.output).toEqual([new Fr(3)]);
     });
 
+    it('Should execute contract function that performs U128 addition', async () => {
+      const calldata: Fr[] = [
+        // First U128
+        new Fr(1),
```

Comment:
If this is the only test, I would welcome to have integer values which fit into U128 but not into other lower integer types such as U64, etc... This way we know this is really working for truly U128 integers.
Just replace the values by: 11111111111....., 22222222.... and it remains very readable.